state_msgs[] group_state